.github/workflows/build-phar.yml

```diff
@@ -0,0 +1,25 @@
+name: Build PHAR
+
+on:
+    - pull_request
+    - push
+
+jobs:
+    phar:
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.0
+                    coverage: none
+
+            -   name: Build PHAR
+                run: |
```

Comment:
later I will want this step to upload phar to github releases (I guess I will do it on my own later). with that, we need to ensure that phar building is executed only after successful tests, I think

Reply:
Done
> ensure that phar building is executed only after successful tests